staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_internal_test.go

```diff
@@ -213,7 +219,10 @@ func TestEnsureInternalLoadBalancerWithExistingResources(t *testing.T) {
 
 	gce, err := fakeGCECloud(vals)
 	require.NoError(t, err)
-	svc := fakeLoadbalancerService(string(LBTypeInternal))
+	svc, err := createFakeLoadbalancerService(string(LBTypeInternal), nil, gce)
+	if err != nil {
```

Comment:
this clashes with the use of require
if you want to use require, then this should be require.NoError(t, err)

Reply:
This test file has a mix of require, assert and if checks for error. I am trying to use if checks in the newer changes I make to this file. Is that ok?

Reply:
can we make it consistent w/in a given test function; otherwise we will never get convergence
also, if we are moving away from `require`, should drop a comment to say where we want to go for future edits